modules/rules_xcodeproj/metadata.json

```diff
@@ -16,13 +16,13 @@
         "github:MobileNativeFoundation/rules_xcodeproj"
     ],
     "versions": [
-        "1.0.0rc1",
-        "1.0.0rc2",
-        "1.0.0rc3",
         "0.12.0",
         "0.12.2",
         "0.12.3",
         "1.0.1",
+        "1.0.0rc1",
```

Comment:
Unfortunately, our current version format will actually put `1.0.0rc1` after `1.0.1`, `1.0.0-rc1` would have worked correctly though.

Reply:
Yeah, it was that way, which is why I had to release 1.0.1.